types/wlr_idle_inhibit_v1.c

```diff
@@ -0,0 +1,185 @@
+#include <assert.h>
+#include <stdlib.h>
+#include <wlr/util/log.h>
+#include <util/signal.h>
+#include <wlr/types/wlr_surface.h>
+#include <wlr/types/wlr_idle_inhibit_v1.h>
+#include "wayland-util.h"
+#include "wayland-server.h"
+#include "idle-inhibit-unstable-v1-protocol.h"
+
+static struct zwp_idle_inhibit_manager_v1_interface idle_inhibit_impl;
+
+static struct zwp_idle_inhibitor_v1_interface idle_inhibitor_impl;
+
+struct wlr_idle_inhibit_manager_v1 *
```

Comment:
Static?

Reply:
Or exported... But should be either one of those.

Do we have a policy on when to export the `wl_resource_get_user_data` wrappers?

Reply:
The policy so far has been "export it if we need it elsewhere". Even if this is reasonable for protocols like this one, we can't guess what users will need. So maybe we should export all of them.